js/components/AppVersionListComponent.jsx

```diff
@@ -134,14 +134,14 @@ var AppVersionListComponent = React.createClass({
         <h5>
           Current Version
           <button className="btn btn-sm btn-info pull-right"
-              onClick={this.handleRefresh}>
+            onClick={this.handleRefresh}>
             ↻ Refresh
           </button>
         </h5>
         <AppVersionComponent
-            app={this.props.app}
-            appVersion={this.props.app.getCurrentVersion()}
-            currentVersion={true} />
+          app={this.props.app}
+          appVersion={this.props.app.getCurrentVersion()}
+          currentVersion={true} />
           {this.getAppVersionTable()}
```

Comment:
Unindent `{this.getAppVersionTable()}` one time, so it gets in line with `<AppVersionComponent`, since it is a sibling.